<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
}

QPushButton{
	color: white;
	height: 30px;
	border: none;
	border-radius: 10px;
}

QPushButton:checked{
	background-color: rgb(245, 250, 254);
	color: #1F95EF;
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/images/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/dashboard_white.png</normaloff>
             <normalon>:/images/dashboard.png</normalon>:/images/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/profile_white.png</normaloff>
             <normalon>:/images/profile.png</normalon>:/images/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messeges_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/messages_white.png</normaloff>
             <normalon>:/images/messages.png</normalon>:/images/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/notifications_white.png</normaloff>
             <normalon>:/images/notifications.png</normalon>:/images/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/settings_white.png</normaloff>
             <normalon>:/images/settings.png</normalon>:/images/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>534</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/images/log_out_white.png</normaloff>:/images/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(31, 149, 239);
	color: white;
}

QPushButton{
	color: white;
	text-align: left;
	height: 30px;
	border: none;
	padding-left: 10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

QPushButton:checked{
	
	background-color: rgb(245, 250, 254);
	color: #1F95EF;
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/images/profile_pic.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>SideBar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_2">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/dashboard_white.png</normaloff>
             <normalon>:/images/dashboard.png</normalon>:/images/dashboard_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_2">
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/profile_white.png</normaloff>
             <normalon>:/images/profile.png</normalon>:/images/profile_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messeges_2">
           <property name="text">
            <string>Messeges</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/messages_white.png</normaloff>
             <normalon>:/images/messages.png</normalon>:/images/messages_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_2">
           <property name="text">
            <string>Notifications</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/notifications_white.png</normaloff>
             <normalon>:/images/notifications.png</normalon>:/images/notifications_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_2">
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/images/settings_white.png</normaloff>
             <normalon>:/images/settings.png</normalon>:/images/settings_white.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>534</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Sign out</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/images/log_out_white.png</normaloff>:/images/log_out_white.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="menu">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>294</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>294</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/image.png</normaloff>:/images/image.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="header_widget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <widget class="QWidget" name="dashboard_page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>240</y>
             <width>361</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Dashboard Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="profile_page">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>280</y>
             <width>361</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Profile Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="messeges_page">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>240</y>
             <width>361</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Messeges Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="notifications_page">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>290</y>
             <width>361</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Notification Page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="settings_page">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>260</y>
             <width>361</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Settings Page</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messeges_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>messeges_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messeges_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>messeges_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>845</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>888</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>850</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>890</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
